.github/workflows/ci.yml

```diff
@@ -47,5 +47,3 @@ jobs:
         run: rebar3 test
       - shell: bash
         run: rebar3 escriptize && _build/default/bin/elvis rock
-      - shell: bash
-        run: rebar3 as test test
```

Comment:
Explained in commit message: `test` (the command) already runs in profile `test`, by default, so this is simply duplicating what is already done above.

Reply:
Are you sure? I know that `rebar3 ct` uses the test profile by default, but `test` is a custom alias… how does rebar3 know that it has to run it on the `test` profile? 🤔

Reply:
I reverted this change. Even if it takes a while longer, it's maybe not inocuous that it's there 😄